### ISSUE TITLE
Camera not working on android [$15]

### ISSUE THREAD

**Issue by nesergen** [kivy/kivy#2831]
When I start my programm on TabDevice the Camera widget shows the white screen. On DescktopComputer the Camera widget works perfect. 

My code:
```
class MyApp(App):
	
	def build(self):
		parent = Widget()

		layout = GridLayout(cols=1, rows =3, spacing=2, size=(800,800), halign="left")
		parent.add_widget(layout)

		cam = Camera(play= True)

		layout.add_widget(cam)

		return parent

if __name__ == '__main__':
	MyApp().run()
```

ErrorLog:
```
[INFO              ] Logger: Record log in /storage/emulated/0/kivy/experim/.kivy/logs/kivy_15-01-08_0.txt
[INFO              ] Kivy v1.8.0
[INFO              ] Factory: 157 symbols loaded
[DEBUG             ] Cache: register <kv.lang> with limit=None, timeout=Nones
[DEBUG             ] Cache: register <kv.image> with limit=None, timeout=60s
[DEBUG             ] Cache: register <kv.atlas> with limit=None, timeout=Nones
[WARNING           ] stderr: /data/data/org.kivy.pygame/files/lib/python2.7/site-packages/kivy/core/image/img_pygame.py:12: RuntimeWarning: import cdrom: No module named cdrom
[WARNING           ] stderr: (ImportError: No module named cdrom)
[INFO              ] Image: Providers: img_tex, img_dds, img_pygame, img_pil, img_gif 
[DEBUG             ] Cache: register <kv.texture> with limit=1000, timeout=60s
[DEBUG             ] Cache: register <kv.shader> with limit=1000, timeout=3600s
[INFO              ] Text: Provider: pygame
[DEBUG             ] Cache: register <kv.loader> with limit=500, timeout=60s
[INFO              ] Loader: using a thread pool of 2 workers
[DEBUG             ] Cache: register <textinput.label> with limit=None, timeout=60.0s
[DEBUG             ] Cache: register <textinput.width> with limit=None, timeout=60.0s
[DEBUG             ] Camera: Ignored <pygst> (import error)
[DEBUG             ] Camera: Ignored <opencv> (import error)
[CRITICAL          ] Camera: Unable to find any valuable Camera provider at all!
[DEBUG             ] App: Loading kv <./my.kv>
[DEBUG             ] App: kv <./my.kv> not found
[DEBUG             ] Window: Ignored <egl_rpi> (import error)
[INFO              ] Window: Provider: pygame(['window_egl_rpi'] ignored)
[DEBUG             ] WinPygame: Set window to fullscreen mode
[DEBUG             ] Window: Display driver android
[DEBUG             ] Window: Actual window size: %dx%d
[INFO              ] GL: OpenGL version <OpenGL ES 2.0 build 1.9@2564405>
[INFO              ] GL: OpenGL vendor <Imagination Technologies>
[INFO              ] GL: OpenGL renderer <PowerVR SGX 544MP>
[INFO              ] GL: OpenGL parsed version: 2, 0
[INFO              ] GL: Texture max size <4096>
[INFO              ] GL: Texture max units <8>
[DEBUG             ] Shader: Fragment compiled successfully
[DEBUG             ] Shader: Vertex compiled successfully
[DEBUG             ] ImagePygame: Load </data/data/org.kivy.pygame/files/lib/python2.7/site-packages/kivy/data/glsl/default.png>
[INFO              ] Support: Android install hooks
[INFO              ] Window: virtual keyboard not allowed, single mode, not docked
[INFO              ] OSC: using <thread> for socket
[DEBUG             ] Base: Create provider from android
[INFO              ] Base: Start application main loop
[INFO              ] Android: found 17 joystick
[INFO              ] Android: create joystick <0>
[INFO              ] Android: discard joystick <0> cause no button
[INFO              ] Android: create joystick <1>
[INFO              ] Android: create joystick <2>
[INFO              ] Android: create joystick <3>
[INFO              ] Android: create joystick <4>
[INFO              ] Android: create joystick <5>
[INFO              ] Android: create joystick <6>
[INFO              ] Android: create joystick <7>
[INFO              ] Android: create joystick <8>
[INFO              ] Android: create joystick <9>
[INFO              ] Android: create joystick <10>
[INFO              ] Android: create joystick <11>
[INFO              ] Android: create joystick <12>
[INFO              ] Android: create joystick <13>
[INFO              ] Android: create joystick <14>
[INFO              ] Android: create joystick <15>
[INFO              ] Android: create joystick <16>
[INFO              ] GL: NPOT texture support is available
[DEBUG             ] Atlas: Load </data/data/org.kivy.pygame/files/lib/python2.7/site-packages/kivy/data/../data/images/defaulttheme.atlas>
[DEBUG             ] Atlas: Need to load 1 images
[DEBUG             ] Atlas: Load </data/data/org.kivy.pygame/files/lib/python2.7/site-packages/kivy/data/../data/images/defaulttheme-0.png>
[DEBUG             ] ImagePygame: Load </data/data/org.kivy.pygame/files/lib/python2.7/site-packages/kivy/data/../data/images/defaulttheme-0.png>

```
ErrorLog when I use resolution parameter:
```
I/python  ( 4045):  Traceback (most recent call last):
I/python  ( 4045):    File "main.py", line 46, in <module>
I/python  ( 4045):      MyApp().run()
I/python  ( 4045):    File "/home/tito/code/python-for-android/build/python-install/lib/python2.7/site-packages/kivy/app.py", line 766, in run
I/python  ( 4045):    File "main.py", line 29, in build
I/python  ( 4045):      cam = Camera(play= True, resolution = (640, 480))
I/python  ( 4045):    File "/home/tito/code/python-for-android/build/python-install/lib/python2.7/site-packages/kivy/uix/camera.py", line 89, in __init__
I/python  ( 4045):    File "/home/tito/code/python-for-android/build/python-install/lib/python2.7/site-packages/kivy/uix/camera.py", line 101, in _on_index
I/python  ( 4045):  TypeError: 'NoneType' object is not callable
E/        (  132): AudioCloseDumpPCMFile file== NULL
E/        (  132): AudioCloseDumpPCMFile file== NULL

```

**Comment by tito** [comment]
There is no Camera support on Android right now

**Comment by jasonxunxu** [comment]
I just posted a PR #3236 to fix this.

**Comment by jasonxunxu** [comment]
Resolved by PR #3250 (merged).
